nusslein_vsrp/latex/overview.tex

```diff
@@ -217,39 +221,42 @@ \section{Introduction}
 the stage values are known, in a way that ensures positivity.
 The selection of the weights requires the solution of a linear program (LP) at every
 step for which the numerical solution would otherwise be non-positive.
-This is a significant cost, but may in fact be an economical alternative to rejecting
+This is a significant cost, but may in some cases be an economical alternative to rejecting
 a step or using excessively small step sizes.
 
 The idea of using different weights within an RKM is not new; for instance it is
 the basis of error approximation using embedded RK pairs \cite{hairer_solving_1993}.
 The idea of adapting the weights after calculating the stage values has also been used,
-for instance in \cite{ketcheson_spatially_2013}.
-In this case it is used to adapt the properties of the time integrator for a method of lines solution of a PDE. With this approach, it is possible to have different properties of the RKM at different parts of the domain.
-Another class of methods that adapt the weights at the end of an RK step
-are the relaxation Runge--Kutta (RRK) methods.
-In these, the weights are scaled by a scalar relaxation parameter in order to guarantee
-conservation or monotonicity of a desired functional; e.g. to conserve or
-dissipate energy or entropy
+for instance in \cite{ketcheson_spatially_2013}.  In this case it is used to
+adapt the properties of the time integrator for a method of lines solution of a
+PDE.  Another class of methods that adapt the weights at the end of an RK step
+are the relaxation Runge--Kutta (RRK) methods.  In these, the weights are
+scaled by a scalar relaxation parameter in order to guarantee conservation or
+monotonicity of a desired functional; e.g. to conserve or dissipate energy or
+entropy
 \cite{ketcheson_relaxation_2019,ranocha_relaxation_2019,ranocha2020general}.
 
 Our means to ensure positivity can be interpreted as a projection
 approach, where the numerical solution is adapted to satisfy the
-positivity constraint at the end of each step. In contrast to the
-usual orthogonal projection, which has also been proposed to deal
+positivity constraint at the end of each step. In contrast to
+simple orthogonal projection, which has also been proposed to deal
 with positivity constraints \cite{shampine1986conservation},
 our approach preserves all linear invariants of the given ODE. These
 invariants can be very important. e.g.\ the total mass for a transport
-problem or in reaction systems. The property to preserve all linear
-invariants is shared by RRK methods and has been shown to be an
-important advantage compared to orthogonal projection methods
+problem or in reaction systems. Preserve of linear
```

Comment:
As a non-native speaker, I would have used "Preservation" instead of "Preserve" (which sounds more like jam to me). Is that also correct?

Reply:
Right; that was a typo.  I've fixed it.